src/mixins/itext_behavior.mixin.ts

```diff
@@ -461,46 +399,21 @@ export abstract class ITextBehaviorMixin<
     this._tick();
     this.fire('editing:entered');
     this._fireSelectionChanged();
-    if (!this.canvas) {
-      return this;
-    }
-    this.canvas.fire('text:editing:entered', { target: this });
-    this.initMouseMoveHandler();
-    this.canvas.requestRenderAll();
-    return this;
-  }
-
-  exitEditingOnOthers(canvas: Canvas) {
-    if (canvas._iTextInstances) {
-      canvas._iTextInstances.forEach((obj) => {
-        obj.selected = false;
-        if (obj.isEditing) {
-          obj.exitEditing();
-        }
-      });
+    if (this.canvas) {
+      this.canvas.fire('text:editing:entered', { target: this });
+      this.canvas.requestRenderAll();
     }
   }
 
   /**
-   * Initializes "mousemove" event handler
-   */
-  initMouseMoveHandler() {
-    this.canvas.on('mouse:move', this.mouseMoveHandler);
-  }
-
-  /**
-   * @private
+   * called by {@link canvas#textEditingManager}
    */
-  mouseMoveHandler(options) {
-    if (!this.__isMousedown || !this.isEditing) {
-      return;
-    }
-
+  updateSelection(e: TPointerEvent) {
```

Comment:
this method will be called only if a registered instance is editing so no need to recheck